packages/server/src/chatgpt.js

```diff
@@ -0,0 +1,284 @@
+const express = require('express');
```

Comment:
Why is this route in the root of server?

Reply:
This file is not in use any more, just keeping it for reference